pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -174,7 +174,8 @@ func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (recon
 		return reconcile.Result{}, nil
 	}
 
-	orgParentFolderID, err := util.GetGCPParentFolderFromConfigMap(r.client, operatorNamespace, orgGcpConfigMap)
+	configmap := util.GetConfigMapOperations(r.client, orgGcpConfigMap, operatorNamespace)
```

Comment:
I find the name ConfigMapOperations confusing when first reading it... It's just a wrapper around the config map right? Shouldn't we call it `GcpProjectOperatorConfigMap` or (less verbose) `OperatorConfig`?

Reply:
replaced with `GcpProjectOperatorConfigMap`